Mathlib/Analysis/SpecialFunctions/Log/Deriv.lean

```diff
@@ -319,4 +319,25 @@ theorem hasSum_log_one_add_inv {a : ℝ} (h : 0 < a) :
     linarith
   · field_simp
 
+/-- Expansion of `log (a + 1)` as a series in powers of `a / (a + 2)`. -/
```

Comment:
```suggestion
/-- Expansion of `log (1 + a)` as a series in powers of `a / (a + 2)`. -/
```
There is another mistake of docstring in the existing part of this file, a few lines above I think, where `hasSum_pow_div_log_of_abs_lt_1` should be `hasSum_pow_div_log_of_abs_lt_one`. If you like, you can fix it as well.